docs/compose-file.md

```diff
@@ -428,9 +428,17 @@ Both `$VARIABLE` and `${VARIABLE}` syntax are supported. Extended shell-style
 features, such as `${VARIABLE-default}` and `${VARIABLE/foo/bar}`, are not
 supported.
 
-If you need to put a literal dollar sign in a configuration value, use a double
-dollar sign (`$$`).
+When you want to set an environment variable that is not to be interpolated by
+Compose, or the configuration value needs a literal dollar sign you can use a
+double dollar sign (`$$`).
 
```

Comment:
@moxiegirl  ah yeh, I see what you're saying and the difference in how it reads. I want to write in a way that's accessible to everyone. I'll update it, thank you for your feedback :smile_cat: